src/keymgmt.c

```diff
@@ -165,7 +165,8 @@ static int p11prov_rsakm_secbits(int bits)
     if (bits > 4096) return 152;
     if (bits > 3072) return 128;
     if (bits > 2048) return 112;
-    if (bits <= 2048) return 0;
```

Comment:
I think I will have to fix these comparisons, they probably should all be >=, returning 0 for 2048 is wrong ...

Reply:
I did this while rebasing and fixing the conflicts, but if you want this as a separate commit, that's fine too.